pyrit/datasets/fetch_example_datasets.py

```diff
@@ -371,3 +371,39 @@ def fetch_harmbench_examples(
     )
 
     return dataset
+
+def fetch_pku_safeRLHF_dataset( 
+    include_safe_prompts: bool = True) -> PromptDataset:
+    """
+    Fetch PKU-SafeRLHF examples and create a PromptDataset.
+
+    Args:
+        include_safe_prompts (bool): If this field is passed as false, we will only grab the unsafe prompts
+
+    Returns:
+        PromptDataset: A PromptDataset containing the examples.
+
+    Note:
+        For more information and access to the original dataset and related materials, visit:
+        https://huggingface.co/datasets/PKU-Alignment/PKU-SafeRLHF
+    """
+    data = load_dataset('PKU-Alignment/PKU-SafeRLHF', 'default')
+    if include_safe_prompts:
+        prompts = [item['prompt'] for item in data['train']]
+    else: 
+        prompts = [item['prompt'] for item in data['train'] if (item['is_response_0_safe'] == False or item['is_response_1_safe'] == False)]
+
+    harm_categories = """Endangering National Security, Insulting Behavior, Discriminatory Behavior, Endangering Public Health, Copyright Issues, Violence, Drugs, 
+    Privacy Violation, Economic Crime, Mental Manipulation, Human Trafficking, Physical Harm, Sexual Content, Cybercrime, Disrupting Public Order, 
+    Environmental Damage, Psychological Harm, White-Collar Crime, Animal Abuse"""
+
+    dataset = PromptDataset(
+        name="PKU-SafeRLHF",
+        description="This is a Hugging Face dataset that labels a prompt and 2 responses categorizing their helpfulness or harmfulness. Only the \'prompt'\ column is extracted.",
+        harm_category=harm_categories,
+        should_be_blocked=not include_safe_prompts,
```

Comment:
```suggestion
        should_be_blocked=True,
```
This is a bit of a tricky piece of metadata as it should be on the prompt level. Let's set it to True for now and I'm revamping the structure of PromptDataset soon anyway.